doc/source/tune/advanced.rst

```diff
@@ -0,0 +1,467 @@
+Advanced Topics, Configurations, & FAQ
```

Comment:
I'm not sure if Advanced Topics is the right name for this. Some of this stuff is pretty core and may not be solely for advanced users.

Reply:
What should I move out?  Do you have a recommendation?

Reply:
I would just rename this file, much of this isn't even advanced concepts - `User Guide and Configurations`?

Reply:
+1 on just keeping it as `User Guide` or `Extended User Guide`

Reply:
done